Newer Ruby and droplet_kit compatibility

scanevari opened the pull request:
faraday-0.13.1 and rfc3986_parser.rb from Ruby 2.4 need encoded droplet id.

scanevari committed:
Newer Ruby and droplet_kit compatibility

faraday-0.13.1 and rfc3986_parser.rb from Ruby 2.4 need encoded droplet id.

scanevari committed:
Version numbers update

Ubuntu
Ruby
HAProxy

scanevari committed:
Merge pull request #1 from scanevari/patch-2

Version numbers update

scanevari committed:
Updated versions + editorial change

thisismitch commented on README.md:
```diff
@@ -13,7 +13,7 @@ The choice of using a basic inventory file was intentional, to provide a simple
-Create an Ubuntu 14.04 Droplet on DigitalOcean. Be sure to choose a region that supports private networking and user data.
+Create an Ubuntu 16.04 Droplet on DigitalOcean. Be sure to choose a region that supports private networking and user data.
 
 [Install HAProxy 1.5](https://www.digitalocean.com/community/tutorials/how-to-implement-ssl-termination-with-haproxy-on-ubuntu-14-04#install-haproxy-15x) on it:
```
This should be HAProxy 1.6 and link to https://www.digitalocean.com/community/tutorials/how-to-implement-ssl-termination-with-haproxy-on-ubuntu-14-04#install-haproxy-16x, right?

thisismitch commented on README.md:
```diff
@@ -13,7 +13,7 @@ The choice of using a basic inventory file was intentional, to provide a simple
 ## Installation
 
-Create an Ubuntu 14.04 Droplet on DigitalOcean. Be sure to choose a region that supports private networking and user data.
+Create an Ubuntu 16.04 Droplet on DigitalOcean. Be sure to choose a region that supports private networking and user data.
```
I would delete the 2nd sentence because all regions support private networking and user data now. And modify the first sentence to mention that the droplet needs Private Networking.

thisismitch commented on doproxy.yml.sample:
```diff
@@ -1,7 +1,7 @@
 ---
-token: 878a490235d53e34b44369b8e78
+token: <YOUR_DO_API_TOKEN>
 ssh_key_ids:           # DigitalOcean ID for your SSH Key
```
Consider deleting this comment.

thisismitch reviewed (changes requested):
The link here https://github.com/thisismitch/doproxy/pull/6/files#diff-04c6e90faac2675aa89e2176d2eec7d8R66 should be https://cloud.digitalocean.com/settings/security.

With the move to Ubutnu 16.04, the nginx default root changed to `/var/www/html`. https://github.com/thisismitch/doproxy/blob/948ed901c2937c15348c1935ee777e155ce49174/user-data.yml.sample#L8 should be updated to be: `echo Droplet: $HOSTNAME, ID: $DROPLET_ID, IP Address: $PUBLIC_IPV4 > /var/www/html/index.html`

scanevari committed:
HAProxy 1.6 / Private networking comments

The link for the installation of HAProxy 1.6 still points to the installation of 1.5 because there is no newer article to show the installation on 1.6.
Since both versions are pretty similar, this should serve as a very helpful guide until a new version of the article is written and the link is updated.

scanevari committed:
Deleted comment

Code is very self explanatory

thisismitch commented on README.md:
```diff
@@ -13,12 +13,12 @@ The choice of using a basic inventory file was intentional, to provide a simple
+Create an Ubuntu 16.04 Droplet on DigitalOcean. The Droplet will need private networking.
 
-[Install HAProxy 1.5](https://www.digitalocean.com/community/tutorials/how-to-implement-ssl-termination-with-haproxy-on-ubuntu-14-04#install-haproxy-15x) on it:
+[Install HAProxy 1.6](https://www.digitalocean.com/community/tutorials/how-to-implement-ssl-termination-with-haproxy-on-ubuntu-14-04#install-haproxy-15x) on it:
```
The anchor should be `16x`.

scanevari committed:
Corrected anchor

scanevari committed:
Changed folder for index.html

scanevari committed:
Corrected link
